packages/firebase_vertexai/firebase_vertexai/example/lib/pages/bidi_page.dart

```diff
@@ -48,11 +48,9 @@ class _BidiPageState extends State<BidiPage> {
   bool _recording = false;
   late LiveGenerativeModel _liveModel;
   late LiveSession _session;
-  final _audioManager = AudioStreamManager();
-  final _audioRecorder = InMemoryAudioRecorder();
-  var _chunkBuilder = BytesBuilder();
-  var _audioIndex = 0;
   StreamController<bool> _stopController = StreamController<bool>();
+  final AudioOutput audioOutput = AudioOutput();
+  final AudioInput audioInput = AudioInput();
```

Comment:
Can these be kept private? I don't see any references outside this library.

Also consider omitting the repeated type name.

```suggestion
  final _audioOutput = AudioOutput();
  final _audioInput = AudioInput();
```